ci/README.md

```diff
@@ -0,0 +1,50 @@
+# Deploying with [Concourse](https://concourse-ci.org/)
+
+To deploy this application with Concourse, you must have a Kubernetes cluster already provisioned and be logged in to a Concourse instance that has access to the cluster.
+
+## Deploying credentials
+
+Before deploying the app you need to create credentials on Kubernetes. This can be done via Concourse using the following task commands:
+
+```sh
+EQ_KEYS_FILE=<path_to_keys_file> \
+EQ_SECRETS_FILE=<path_to_secrets_file> \
+PROJECT_ID=<project_id> \
+REGION=<cluster_region> \
+fly -t <target-concourse> execute \
+  --config ci/deploy_credentials.yaml \
+  --input eq-questionnaire-runner-repo=.
```

Comment:
Did you try renaming the input to the repo name and omitting the `--input`?

Reply:
Yes - fly execute worked without specifying the input flag when the name is the same as the cwd

Reply:
So, the `--input` can be removed? 😉